docs/release-notes/2.38.0.md

```diff
@@ -0,0 +1,149 @@
+# Percona Monitoring and Management 2.38.0
+
+| **Release date:** | JUn 02, 2023                                                                                    |
+| ----------------- | ----------------------------------------------------------------------------------------------- |
+| **Installation:** | [Installing Percona Monitoring and Management](https://www.percona.com/software/pmm/quickstart) |
+
+Percona Monitoring and Management (PMM) is an open source database monitoring, management, and observability solution for MySQL, PostgreSQL, and MongoDB.
+
+<!---
+!!! caution alert alert-warning "Important/Caution"
+    Crucial points that need emphasis:
+
+    - Important: A significant point that deserves emphasis.
+    - Caution: Used to mean 'Continue with care'.
+ --->
+
+
+## Release Highlights
+
+### Important security updates
+
+- As security is paramount to us, starting with PMM 2.38.0, we have laid a solid and modernized enterprise-grade foundation for PMM. We have migrated our base operating system on which we build our base image from Enterprise Linux 7 ( CentOS 7) to Enterprise Linux 9 (EL9). Thus starting with PMM 2.38.0, we have made PMM publicly available on EL9 (Oracle Linux 9) as an early preview.
+
+    The benefits of this migration are manifold. The migration fixes a large number of CVEs and improves upstream's response time to security-related issues. You can access more modern libraries to build on long with better compatibility with new technologies. 
+
+    We will be rolling this out in phases. 
+
+    The following users will be able to take advantage of this immediately:
+
+    - Brand new installations of PMM, regardless of how you choose to run it
+        * Docker: substitute perconalab/pmm-server:2.37.0-el9 in the docker run commands
+        * OVF: [Download](https://downloads.percona.com/downloads/pmm2/2.37.0/ova/pmm-server-2.37.0-el9.ova) it directly from percona.com.
+    - Docker/Podman users that do full container replacement upgrades to newer versions of PMM*
+        * substitute perconalab/pmm-server:2.37.0-el9 in the docker run commands
+
+    Users who will need to wait a bit longer:
+
+    - AMI users
+    - Users that exclusively upgrade through the UI’s “Upgrade Now” button
+    - OVF users
+
+- Components upgrade - Grafana has been upgraded to 9.2.18 due to [CVE-2023-28119](https://github.com/advisories/GHSA-5mqj-xc49-246p) and [CVE-2023-1387](https://github.com/advisories/GHSA-c3h9-vpfv-3x4m).
+
+- Exporters - Updated the mysqld_exporter, node_exporter, postgres_exporter, mongo_exporter to fix the CVE's.
```

Comment:
@BupycHuk can you pls recheck what exactly was updated in this release for the exporters?